src/workers/python/python_package.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+set -eux
+
+rm -rf ./package ./package.tar
+mkdir ./package
+pip install -t ./package python-runner friendly_traceback jedi
+cp -r ./papyros ./package
+cd ./package
+tar cfv ./../package.tar *
```

Comment:
I believe npm compresses the bundle, so I don't think I need to add something here. Otherwise the webpack loader also needs a decompressing step, which might not be supported.

Reply:
Have you tried this compressing yourself? I have tried different variations to compress the tar into a tar.gz, using python or something like tar -czf, but I keep getting the following error:
```
comlink.mjs:69 Uncaught (in promise) PythonError: Traceback (most recent call last):
  File "/lib/python3.9/shutil.py", line 1184, in _unpack_tarfile
    tarobj = tarfile.open(filename)
  File "/lib/python3.9/tarfile.py", line 1616, in open
    raise ReadError("file could not be opened successfully")
tarfile.ReadError: file could not be opened successfully

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/lib/python3.9/site-packages/pyodide/_util.py", line 33, in unpack_buffer_archive
    shutil.unpack_archive(f.name, extract_dir, format)
  File "/lib/python3.9/shutil.py", line 1247, in unpack_archive
    func(filename, extract_dir, **dict(format_info[2]))
  File "/lib/python3.9/shutil.py", line 1186, in _unpack_tarfile
    raise ReadError(
shutil.ReadError: /tmp/tmpm0qlluh_ is not a compressed or uncompressed tar file

    at new_error (pyodide.asm.js:14:247183)
    at pyodide.asm.wasm
    at pyodide.asm.wasm
    at Object.Module._pythonexc2js (pyodide.asm.js:14:944230)
    at Object.Module.callPyObjectKwargs (pyodide.js:1:10739)
    at Proxy.callKwargs (pyodide.js:1:18742)
    at Object.unpackArchive (pyodide.js:1:24012)
    at eval (PythonWorker.worker.ts?./node_modules/ts-loader/index.js:76:29)
    at step (PythonWorker.worker.ts?./node_modules/ts-loader/index.js:56:23)
    at Object.eval [as next] (PythonWorker.worker.ts?./node_modules/ts-loader/index.js:37:53)
```
Manually calling shutil.unpack_archive gives me the files, so something must be going wrong when using url-loader to obtain the tar.gz contents. Does this sound familiar to you, or do you have a similar experience?